### PR TITLE
Openpype settings project global

### PR DESCRIPTION
**Pull request by SherineKB**
## Changelog Description
Update of the Project Global settings : 
Re-organized the page, added the missing categories of the panel and completed their informations.

### CODE REVIEW

File: website/docs/admin_settings_local.md [hunk 1]
```diff
@@ -16,13 +16,21 @@ OpenPype stores some of it's settings and configuration in local file system. Th
...
 ### OpenPype Mongo URL
+The **Mongo URL** is the database URL given by your Studio. More details [here](artist_getting_started.md#mongodb).
 
 ### General
+**OpenPype Username** : enter your username (it can also take by default the computer session username). It signs your actions on **OpenPype**.
+
+**Admin permissions** : on checked, no need to enter a password (if defined) to access to the **Admin** section.
 
 ### Experimental tools
+Futur version of existing tools or new ones.
```
Comment (BigRoy) [inline website/docs/admin_settings_local.md:27]:
```suggestion
Future version of existing tools or new ones.
```

File: website/docs/admin_settings_local.md [hunk 2]
```diff
@@ -16,13 +16,21 @@ OpenPype stores some of it's settings and configuration in local file system. Th
 ## Categories
 
 ### OpenPype Mongo URL
+The **Mongo URL** is the database URL given by your Studio. More details [here](artist_getting_started.md#mongodb).
 
 ### General
+**OpenPype Username** : enter your username (it can also take by default the computer session username). It signs your actions on **OpenPype**.
```
Comment (BigRoy) [inline website/docs/admin_settings_local.md:22]: Admittedly this is unclear to me. How does it sign my actions on OpenPype? What does that influence?

Here is it a bit tweaked. Note that I'm not sure if my example is actually correct and whether it does that.

```suggestion
**OpenPype Username** : enter your username (if not provided, it uses computer session username by default). This username is used to sign your actions on **OpenPype**, for example the "author" on a publish.
```

File: website/docs/admin_settings_local.md [hunk 3]
```diff
@@ -16,13 +16,21 @@ OpenPype stores some of it's settings and configuration in local file system. Th
...
+
+**Admin permissions** : on checked, no need to enter a password (if defined) to access to the **Admin** section.
```
Comment (BigRoy) [inline website/docs/admin_settings_local.md:24]:
```suggestion
**Admin permissions** : When enabled you do not need to enter a password (if defined in Studio Settings) to access to the **Admin** section.
```

File: website/docs/admin_settings_local.md [hunk 4]
```diff
@@ -16,13 +16,21 @@ OpenPype stores some of it's settings and configuration in local file system. Th
...
 
 ### Experimental tools
+Futur version of existing tools or new ones.
+
+### Environments
+Local replacement of the environment data of each software and additional internal data necessary to be loaded correctly.
 
 ### Applications
+Local override of software paths and their versions. More details [here](admin_settings_system.md#applications).
```
Comment (BigRoy) [inline website/docs/admin_settings_local.md:33]:
```suggestion
Local override of software executable paths for each version. More details [here](admin_settings_system.md#applications).
```

**Review by ClementHector — CHANGES_REQUESTED** [review]

**Pull request closed without merging**